bigquery/schema_builder.py

```diff
@@ -126,7 +126,7 @@ def bigquery_type(o, timestamp_parser=default_timestamp_parser):
     """
 
     t = type(o)
-    if t == int:
+    if t == int or t == long:
```

Comment:
This doesn't work in python 3 (http://python3porting.com/differences.html#long). Probably should do a check using `six` similar to the one below using `six.integer_types` (https://pythonhosted.org/six/#six.integer_types).